src/ansys/mapdl/core/_commands/aux2_/bin_manip.py

```diff
@@ -21,323 +21,315 @@
-        working directory. Later on, you can start a new distributed memory
-        parallel solution and use the COMBINE command to combine local files
-        into a global file for a downstream solution or another operation
-        (e.g., postprocessing with /POST1). For example, the command
-        COMBINE,RST will combine local results files (JobnameN.RST) into a
-        global results file (Jobname.RST).
-
-        When the COMBINE command is used in a subsequent Distributed ANSYS
-        session, the number of processors must be the same as in the
-        distributed memory parallel solution that generated the files.
-
-        When running on a cluster, the local solution files must be available
-        in the working directory on each node in the subsequent session. As an
-        example, consider the following command line used to generate local
-        solution files:
-
-        Different machines can be used in the subsequent session to combine
-        these files. However, the total number of cores must remain unchanged
-        (seven in the above case), and the local files must be copied to the
-        working directory (or directories) on each of the machines used in the
-        subsequent session.
+        This command copies a matrix from the assembled global matrix file ( :file:`.full` file) or from the
+        superelement matrix file ( :file:`.sub` file) as specified on the :ref:`fileaux2` command and write
+        it in Harwell-Boeing format to a new file named :file:`Jobname.matrix`. The Harwell-Boeing format is
+        widely used by other applications that deal with matrices.
+
+        The assembled global matrix file is created during solution depending on the analysis type, equation
+        solver, and other solution options. By default, the assembled global matrix file is never deleted at
+        the end of solution. For most analysis types, the Sparse direct solver and the ICCG solver will
+        write a :file:`.full` file. All mode extraction methods used for buckling and modal analyses will
+        write a properly formatted :file:`.full` file to be used with the :ref:`hbmat` command. However,
+        when using distributed-memory parallel (DMP) processing, a majority of analyses will write a
+        distributed (or local) form of the
+
+         :file:`.full` file which is not currently supported by the :ref:`hbmat` command.
+
+        When dumping the stiffness matrix for transient and harmonic analyses, be aware that the element
+        mass matrix values (and possibly element damping matrix values) are incorporated into the globally
+        assembled stiffness matrix. Thus, the globally assembled stiffness matrix represents more than the
+        stiffness of the model for these analysis types. Please refer to the `Theory Reference
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_thry/thy_biblio.html>`_  Mechanical APDL
```

Comment:
Parsing error?

Reply:
I am not following, what are you referring to here?

Reply:
I got it now, sorry for the delay 😄 
I am investigating!

Reply:
Fixed in https://github.com/ansys/pyconverter-xml2py/pull/374/commits/75670e3e1d7a1be75217f753d98250bf4bb6db9b